src/css/webflow.css

```diff
@@ -1807,3 +1807,6 @@ textarea.w-select {
 .w-condition-invisible {
   display: none !important;
 }
+#dev-tools {
```

Comment:
This will need to be added on the webflow side, right?

Reply:
yes indeed. I could add it to the svelte component but I like it to be not opinionated how you would use it.